ember-bootstrap/index.js

```diff
@@ -60,32 +54,14 @@ module.exports = {
     this.bootstrapOptions = options;
 
     this.validateDependencies();
-    this.preprocessor = this.findPreprocessor();
-
-    // static Bootstrap CSS is mapped to vendor tree, independent of BS version, so import from there
-    let vendorPath = path.join('vendor', 'ember-bootstrap');
-
-    if (!this.hasPreprocessor()) {
-      // / Import css from bootstrap
-      if (options.importBootstrapCSS) {
-        this.warn(`\
-Importing Bootstrap CSS through Ember Bootstrap is deprecated. Applications should import Bootstrap's CSS explicitly.
-Please find information how to do so in the Ember guides: https://guides.emberjs.com/release/addons-and-dependencies/#toc_css.
-Additionally set importBootstrapCSS configuration of Ember Bootstrap in your app's ember-cli-build.js to false. Please find more information about Ember Bootstrap's configuration here: https://www.ember-bootstrap.com/getting-started/setup
-
-Until: 7.0.0
-`);
-
-        this.needsBootstrapStyles = true;
-        this.import(path.join(vendorPath, 'bootstrap.css'));
-        this.import(path.join(vendorPath, 'bootstrap.css.map'), {
-          destDir: 'assets',
-        });
-      }
-    }
 
-    // import custom addon CSS
-    this.import(path.join(vendorPath, `bs${options.bootstrapVersion}.css`));
+    // import custom addon CSS required to use Popper.js v2 with Bootstrap v4
+    if (options.bootstrapVersion === 4) {
+      let vendorPath = path.join('vendor', 'ember-bootstrap');
+
+      // import custom addon CSS
+      this.import(path.join(vendorPath, `bs4-overwrites-for-popper-js-v2.css`));
```

Comment:
I renamed the file to clarify why we have it. When migrating to a v2 addon, we must request the consumers to import it explicitly in their apps. I added that to the tracking issue for v7.